src/components/folder/mainHeader.js

```diff
@@ -23,21 +31,37 @@ function MainHeader({ activeFolderId, onFolderClick }) {
   };
 
   return (
-    <FolderLinkHeader>
-      <MainNav>
-        <FolderList folderList={folderList} activeFolderId={activeFolderId} onFolderClick={handleFolderClick} />
-        <Link to="#">
-          <FolderAdd>
-            폴더 추가 <FolderAddIcon src={addIcon} alt="폴더 추가 아이콘" />
-          </FolderAdd>
-        </Link>
-      </MainNav>
-
-      <HeaderTitle>
-        <MainTitle>{activeFolderName || '전체'}</MainTitle>
-        <HeaderButtons>{activeFolderName && MAIN_ICON_BUTTONS.map(text => <MainIconButton key={text} text={text} />)}</HeaderButtons>
-      </HeaderTitle>
-    </FolderLinkHeader>
+    <>
+      <FolderLinkHeader>
+        <MainNav>
+          <FolderList folderList={folderList} activeFolderId={activeFolderId} onFolderClick={handleFolderClick} />
+          <Link to="#">
+            <FolderAdd>
+              폴더 추가 <FolderAddIcon src={addIcon} alt="폴더 추가 아이콘" />
+            </FolderAdd>
+          </Link>
+        </MainNav>
+
+        <HeaderTitle>
+          <MainTitle>{activeFolderName || '전체'}</MainTitle>
```

Comment:
```suggestion
          <MainTitle>{activeFolderName ?? '전체'}</MainTitle>
```
[중요도: 중간] 위 방식을 더 권장 드립니다.
|| 연산자: falsy한 값을 탐색함(undefined, null, false, 0 등)
?? 연산자: nullish한 값을 탐색함(undefined, null)

현재 방식은 ||도 가능은 하나 최대한 ??를 사용하는게 괜한 사이드 이펙트 방지에 도움이 됩니다!

Reply:
### 널 병합 할당
[널 병합 할당 (en-US)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)

- ||는 첫 번째 truthy 값을 반환
- ??는 첫 번째 정의된(defined) 값을 반환

```javascript
let height = 0;

alert(height || 100); // 100
alert(height ?? 100); // 0
```
??의 경우 정확히 null이나 undefined인 경우에만 사용

좋은것 배웠습니다. 감사합니다.

Reply:
저보다 더 잘 정리해주셧군요 좋습니다 ㅎㅎ